### PR TITLE
Add release notes for 0.123

### PR DESCRIPTION
**Pull request by joy-yao**

### CODE REVIEW

**Comment by haozhun** [comment]
@electrum Can you review it please? I don't really know how to review release notes.

File: presto-docs/src/main/sphinx/release/release-0.123.rst [hunk 1]
```diff
@@ -6,6 +6,8 @@ General Changes
 ---------------
 
 * Remove ``node-scheduler.location-aware-scheduling-enabled`` config.
+* Optimize execution of cross join.
+* Implement add column support in Raptor.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.123.rst]: Remove this, we don't have release notes for Raptor

**Comment by electrum** [comment]
Looks good

**Comment by electrum** [comment]
Will merge

**Pull request closed without merging**